source/system/Character.ooc

```diff
@@ -6,7 +6,7 @@
  * of the MIT license.  See the LICENSE file for details.
  */
 
-include stdlib
+include stdlib, stdint
```

Comment:
Ping. Will we need to add this file everywhere we use 64-bit integers in order to run on windows?